crawl-ref/source/dat/strings/de/adjectives-e.txt

```diff
@@ -533,10 +533,10 @@
 # these ones are a jewellery material and a rune
 %%%%
 "gossamer "
-"Spinnfäden-"
+"hauchdünne "
```

Comment:
I'm not sure about this correction.

"Gossamer" can metaphorically mean "really thin", but in this case, it means it's literally made of gossamer (i.e. spider silk). At least, that's my understanding.

Reply:
I agree with your understanding. Tried to get rid of the hyphen. We could go with 'Spinnfadenrune' or 'Gespinstrune', but those don't sound great, and I assume there has to be either a hyphen or a space? What about 'seidige Rune' (silky rune)?

Reply:
Actually, I just realised that these rune adjectives are redundant because we have defined a translation for the full string "the gossamer rune" in items-singular-def-nom.txt. I think I was previously defining a translation for "the %s rune", which meant I did need to the adjectives, but no longer.

All we need is the long version in items-singular-def-nom.txt and the terse version "gossamer" (with no trailing space) in items-singular-def-nom.txt. The terse version is a standlone thing rather than an adjective. It's used on the character overview screen like this: "}: 2/15 runes: decaying, gossamer".

So, we're good, we can do "the gossamer rune" = "die Spinnfadenrune" and "gossamer" = "Spinnfaden". We can get rid of "gossamer " and all the other adjectives that are only used for runes.

Reply:
All right, I'll go through the items next and push a commit for them.